torch/nn/modules/batchnorm.py

```diff
@@ -100,10 +100,19 @@ def forward(self, input):
                 else:  # use exponential moving average
                     exponential_average_factor = self.momentum
 
+        # Handle the different cases for the `training` value to forward to F.batch_norm
+        if self.track_running_stats:
+            # If BN stats are tracked, we want to update them only in training mode when the buffers
+            # are not None. If they are passed when None, they will not be updated by F.batch_norm
```

Comment:
maybe you mean "passed *as* None"?

Reply:
Indeed, fixed the typo, thanks! I don't know why flake fails though.